Update and restructure of hackweek roles pages (#35)
uwhackweek/hackweeks-as-a-service: docs/services/lead-community.md

# Lead Hackweek Coordinator for Community

> The Lead Hackweek Coordinator for Community guides the activities of community members engaged in hackweek organization. This is typically the person who contacted eScience requesting a hackweek.

Reviewer: ```suggestion
The Lead Hackweek Coordinator for Community guides the activities of community members engaged in hackweek organization and execution. This is typically the person who contacted eScience requesting a hackweek.
```

## Goals

* Matching the technical and scientific needs of the community to the design of the hackweek.
* Ensuring everyone involved is sufficiently resourced to carry out the work.

> * Strategizing on how best to implement practices of open source science for this community. 

Reviewer: ```suggestion
* Strategizing on how best to implement practices of open source science within this community. 
```

## Specific Activities

> * Acquire funding and develop a budget to support hackweek activities.
> * Work with the community to define the overarching goals of the hackweek.
> * Recruit people to join the hackweek organizing team.
> * Attend the event and be available to assist with project and tutorial work.

Reviewer: ```suggestion
* Acquire funding to support hackweek activities.
* Work with the community to define the overarching goals of the hackweek.
* Build a diverse, inclusive, and equitable hackweek organizing team; recruit broadly for organizers and participants.
* Help organize and attend event planning meetings.
* Review tutorial content, evaluating meeting hackweek goals, learning objectives, and community relevance.
* Attend the hackweek and be available to assist with project and tutorial work.
* Facilitate and/or lead longer-term community efforts to build on collaborations fostered around the hackweek.
```
